angularjs-portal-home/src/main/webapp/my-app/layout/widget/partials/weather.html

```diff
@@ -1,10 +1,7 @@
 
 
 <div class="weather-dropdown">
-  <a class="weather-not-clicked" ng-show="showMetric" href=""
-    ng-click="showMetric=false; changePref('F')">Change to &degF</a>
-  <a class=" weather-not-clicked" ng-show="!showMetric" href=""
-    ng-click="showMetric=true; changePref('C')">Change to &degC</a>
+  <a class=" weather-not-clicked" href="" ng-click="cycleUnits()">Change units to &deg{{nextUnits}}</a>
```

Comment:
This is cute, adding more unit type options while incurring now bigger a UI footprint, arguably no more complicated a UI footprint.